packages/gasket-data-react/README.md

```diff
@@ -0,0 +1,109 @@
+# @gasket/data-react
+
+Helper package for accessing embedded Gasket Data in the browser when using react and NextJS.
+
+## Installation
+
+```
+npm i @gasket/data-react
+```
+
+## Usage
+
+This helper is intended for use in conjunction with @gasket/data and assists with SSR for react apps.
+
+
+### Usage
+#### Example 1
+```js
+// pages/_app.js
+const Root = ({ Page, pageProps }) => {
+  return (
+    <Page {...pageProps} />
+  );
+};
+
+export default withGasketData(Root);
```

Comment:
I think this needs an explanation that it injects the context so the component can consume it later on.